hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -63,367 +63,366 @@ handleMessage HydraNode{eq} = putEvent eq . NetworkEvent defaultTTL
-                waitUntil [n1, n2] ReadyToFanout
-                send n1 Fanout
-                send n2 Fanout
-                waitUntil [n1, n2] $ HeadIsFinalized (utxoRefs [1, 2])
-                allTxs <- reverse <$> readTVarIO (history chain)
-                length (filter matchFanout allTxs) `shouldBe` 2
-
-      it "contest automatically when detecting closing with old snapshot" $
-        shouldRunInSim $ do
-          withSimulatedChainAndNetwork $ \chain ->
-            withHydraNode aliceSk [bob] chain $ \n1 ->
-              withHydraNode bobSk [alice] chain $ \n2 -> do
-                openHead n1 n2
-
-                -- Perform a transaction to produce the latest snapshot, number 1
-                let tx = aValidTx 42
-                send n2 (NewTx tx)
-                waitUntilMatch [n1, n2] $ \case
-                  SnapshotConfirmed{snapshot = Snapshot{number}} -> number == 1
-                  _ -> False
-
-                -- Have n1 & n2 observe a close with not the latest snapshot
-                let deadline = arbitrary `generateWith` 42
-                chainEvent n1 (Observation (OnCloseTx 0 deadline))
-                chainEvent n2 (Observation (OnCloseTx 0 deadline))
-
-                waitUntilMatch [n1, n2] $ \case
-                  HeadIsClosed{snapshotNumber} -> snapshotNumber == 0
-                  _ -> False
-
-                -- Expect n1 to contest with latest snapshot, number 1
-                waitUntil [n1, n2] HeadIsContested{snapshotNumber = 1}
-
-    describe "Hydra Node Logging" $ do
-      it "traces processing of events" $ do
-        let result = runSimTrace $ do
-              withSimulatedChainAndNetwork $ \chain ->
-                withHydraNode aliceSk [] chain $ \n1 -> do
-                  send n1 (Init testContestationPeriod)
-                  waitUntil [n1] $ ReadyToCommit (fromList [alice])
-                  send n1 (Commit (utxoRef 1))
-
-            logs = selectTraceEventsDynamic @_ @(HydraNodeLog SimpleTx) result
-
-        logs
-          `shouldContain` [BeginEvent alice $ ClientEvent $ Init testContestationPeriod]
-        logs
-          `shouldContain` [EndEvent alice $ ClientEvent $ Init testContestationPeriod]
-
-      it "traces handling of effects" $ do
-        let result = runSimTrace $ do
-              withSimulatedChainAndNetwork $ \chain ->
-                withHydraNode aliceSk [] chain $ \n1 -> do
-                  send n1 (Init testContestationPeriod)
-                  waitUntil [n1] $ ReadyToCommit (fromList [alice])
-                  send n1 (Commit (utxoRef 1))
-
-            logs = selectTraceEventsDynamic @_ @(HydraNodeLog SimpleTx) result
-
-        logs `shouldContain` [BeginEffect alice (ClientEffect $ ReadyToCommit $ fromList [alice])]
-        logs `shouldContain` [EndEffect alice (ClientEffect $ ReadyToCommit $ fromList [alice])]
-
-      roundtripAndGoldenSpecs (Proxy @(HydraNodeLog SimpleTx))
```

Comment:
This file is impossible to review. I will continue under the assumption that you only changed the first line @abailly-iohk

Reply:
Yeah, sorry for that. Did you try ignoring WS change?

Reply:
I can confirm that ignoring WS shows a single line change :)